examples/route_guide/client/client.go

```diff
@@ -71,7 +71,7 @@ func printFeatures(client pb.RouteGuideClient, rect *pb.Rectangle) {
 			break
 		}
 		if err != nil {
-			log.Fatalf("%v.ListFeatures(_) = _, %v", client, err)
```

Comment:
I think this is not really wrong as-is.  It prints the name of the RPC and the error returned by that RPC (which is achieved by calling `Recv`).

What exactly is printed when you print the stream with `%v` like this?  If it has the remote target name / IP address and also the method name, it might be interesting to print instead.

Reply:
@dfawley PTAL

Reply:
Now it's even less clear what happened when this occurs.  `stream.Recv failed` is not as descriptive as it doesn't include the name of the RPC in progress.  An error that occurs here is more an error of the RPC itself than the `stream.Recv` operation within the stream.

Can you say:

1. why you believe what we had before was bad,
2. what it looked like before and what the new output will look like (ideally with examples), and
3. why you believe the new way is better?

Reply:
1. It prints pointer when the error returned.
2. It look like `&{0xc0001a8700}.ListFeatures(_) = _, some err`.
    The new output will look like `stream.Recv failed: some err`
3. The new way will not prints pointer.

Reply:
Thanks for clarifying.